src/pages/api/announcements/create.ts

```diff
@@ -0,0 +1,19 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { createAnnouncement } from "../../../data/annoucementData";
```

Comment:
NIT: we have alias for src directory setup (`@`) so this should be `@/data/announcements` same thing on every other page